frontend/src/app/trees/_services/christmas-trees-application.service.ts

```diff
@@ -22,7 +22,20 @@ export class ChristmasTreesApplicationService {
     }
     const options = new RequestOptions({ headers: headers });
 
-    return this.http.post(this.endpoint, body, options).map((res: Response) => res.json()).catch(this.util.handleError);
+    return this.http
+      .post(this.endpoint, body, options)
+      .map((res: Response) => res.json())
+      .catch(this.util.handleError);
+  }
+
+  cancelOldApp(id) {
```

Comment:
this maybe doesn't matter yet, but will this effectively be the failure url?

Reply:
@lauraGgit there is no failure URL. This function cancels the old application when a user returns to the application form after clicking `cancel` at pay.gov, because each time a user fills out the application form, a new application is created.